install/README.md

```diff
@@ -0,0 +1,75 @@
+<!--
+   # @@@ START COPYRIGHT @@@
+   #
+   # Licensed to the Apache Software Foundation (ASF) under one
+   # or more contributor license agreements.  See the NOTICE file
+   # distributed with this work for additional information
+   # regarding copyright ownership.  The ASF licenses this file
+   # to you under the Apache License, Version 2.0 (the
+   # "License"); you may not use this file except in compliance
+   # with the License.  You may obtain a copy of the License at
+   #
+   #   http://www.apache.org/licenses/LICENSE-2.0
+   #
+   # Unless required by applicable law or agreed to in writing,
+   # software distributed under the License is distributed on an
+   # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   # KIND, either express or implied.  See the License for the
+   # specific language governing permissions and limitations
+   # under the License.
+   #
+   # @@@ END COPYRIGHT @@@
+ -->
+
+## Trafodion Installers
+
+* **install** - This is the current command-line installer. It installs a server tarball
+  on an existing CDH or HDP Hadoop cluster.
+* **python-installer** - This is the new command-line installer, meant to replace current
+  command-line installer. Likewise, installs server tarball on existing CDH, HDP,
+  or APACHE cluster.
+* **ambari-installer** - This integrates with Ambari cluster manager, so only applies to HDP.
+  In this case, trafodion server is installed via RPM. This is installed on Ambari server as
+  a management pack. Trafodion can be included in the initial cluster creation or added later.
+
+## Ambari Integration
+
+The Ambari MPack (management pack) is also packaged as an RPM, having a dependency on ambari-server.
+Given a proper yum repo file, `traf_ambari` rpm
+can be installed directly and it pulls in ambari-server.
+If ambari-server is previously installed and running, it must be restarted to pick up the Trafodion
+management pack.
+
+#### Packaging
+
+Part of Ambari's job is to set up yum repo files on each node in order to install packages.
+The default URLs are for Hortonworks' public repos. But since your custom-built Trafodion is
+not hosted there, you need to specify a URL for your local yum repo server. To build that into
+the `traf_ambari` package, use make to specify value of `REPO_URL`.
+
+   `make package REPO_URL=http://my.repo.server/repo/...`
```

Comment:
@mkby, I was just testing again today and found what I said earlier is incorrect. Ambari does give a chance to edit the URL in the GUI.
![image](https://cloud.githubusercontent.com/assets/5273190/21731185/fbe310ce-d407-11e6-9785-2ab56105b015.png)

So that helps a lot.